packages/airnode-deployer/src/utils/infrastructure.ts

```diff
@@ -34,3 +35,101 @@ export class MultiMessageError extends Error {
     super(messages.join('\n'));
   }
 }
+
+export const BUCKET_NAME_REGEX = /^airnode-[a-f0-9]{12}$/;
+export const generateBucketName = () => `airnode-${randomBytes(6).toString('hex')}`;
```

Comment:
> 1. This is to address the GCP unique bucket names limitation right? (I'd prefer a comment for this, since this idea will get forgotten in time).

Sure.

>     2. The bucket name feels a bit awkward when user checks the buckets in AWS. I don't have a better idea though. Maybe I'd call it `airnodes` (plural) since it will hold multiple.
> 
>     3. The common bucket name needs to be documented in the docs. I'd also prefer if the deployer CLI told me that it's creating a shared bucket called `airnodes-xyz`. Same if it's reusing a bucket...

It does tell you that in the debug mode. I don't think users care about this much that's why I put it there. Serverless doesn't provide such info by default I think.